src/components/Navbar/Navbar.jsx

```diff
@@ -127,14 +128,20 @@ const Navbar = () => {
                 ) : (
                   <>
                     <Button
-                      variant="outline"
+                      variant="Primary"
                       to="/auth/login"
                       fontweight="bold"
                     >
-                      Sign in
+                      <span>
+                        Contribute to Milan
+                        <IoChevronForwardSharp style={{ marginLeft: "5px" }} />
+                      </span>
                     </Button>
```

Comment:
As discussed, for now let us hide this

Reply:
@pranay911 , you said that you would hide it - but it is still visible.
![image](https://github.com/milancommunity/Milan/assets/72851613/16825a6b-bcdf-4f7c-885c-898d9f2e7925)

Also when hovering over the button everything disappears, add a border for the hovering instance. Also please make the navbar texts be of a lesser font-weight, 400 would be great.
![image](https://github.com/milancommunity/Milan/assets/72851613/44e9e5fa-ea4e-4fa2-b264-d384362fbfe4)